### DISCUSSION TITLE
Icon

### DISCUSSION THREAD

**Discussion by davidfurdos** [Buduf/PTI-Ear-Trainer#17] [Show and tell]
I have created an icon for the program and also I have set that if you run the code. You can check that below.
![icon](https://user-images.githubusercontent.com/108064994/199715504-318cc944-fbdc-4029-9d73-65c7165bb9cc.jpg)

**Comment by Buduf** [comment]
It's great! But I hope you have a version without the checkered background. 😃

**Reply by davidfurdos** [reply]
Of course, I have, you can check it from now :'D
